### ISSUE TITLE
Higher power consumption after flashing custom kernel

### ISSUE THREAD

**Issue by hoderer** [ClusterM/hakchi2#496]
User SQZ found out on gbatemp that there is a higher power consumption after flashing the custom kernel. 

He wrote: "I made some checks with my AMP/Watt meter: With original kernel: consumption is about 2 Watts. With custom kernel: consumprion is about 3.5 Watts". Before flashing the custom kernel the device can be powered through the usb port of the TV which isn't possible anymore with a custom kernel. The device gets a whole lot hotter, too.

I'm not sure if there's any solution. Just want to let you know. Maybe someone can investigate this. Thanks alot!

**Comment by roflcopter777** [comment]
Here's the source to the post: 
https://gbatemp.net/threads/hakchi2-nes-mini-very-simple-pimp-tool.456256/page-659#post-7621675

What's weird though is I have it plugged into my TV's USB port and I can power it on fine with the custom kernel, so something's definitely weird going on.

**Comment by hoderer** [comment]
there are a lot of different TVs out there. mine is a samsung from 2009, maybe it has a weaker usb power supply than yours.

**Comment by roflcopter777** [comment]
But the fact people are now reporting higher temps in their systems, excessive heat over a prolonged time is an electronic's worst enemy, Maybe the fact the program isn't quite complete yet somehow has to do with the way the custom kernel is affected when flashed, this doesn't look good >.>

**Comment by nowhammies** [comment]
Sounds like some thing as simple as polling vs interrupt behavior

**Comment by codex785** [comment]
Is there a similar issue with nes mini?

**Comment by tecteun** [comment]
@ClusterM maybe related to  the while, in https://github.com/ClusterM/hakchi2/blob/HEAD/mods/mod_hakchi/hakchi/rootfs/bin/remote-exec ?

**Comment by roflcopter777** [comment]
I see, so uh, what do we do? Has anyone else reported warmer temps?

**Comment by tecteun** [comment]
Uncomment the sleep line, move it after the if's, and see if that helps ;)

**Comment by roflcopter777** [comment]
I don't know to do that, someone's gonna have to compile it.

**Comment by sks316** [comment]
The GBAtemp user has said that it's an issue with https://github.com/ClusterM/hakchi2/blob/HEAD/mods/mod_hakchi/hakchi/rootfs/bin/remote-exec and that disabling it seemed to fix the issue. @ClusterM

**Comment by ClusterM** [comment]
Fixed. Please update and reflash custom kernel.

**Comment by Moltdar** [comment]
thanks a lot cluster! are you going to issue/publish minor releases for such fixes?

**Comment by roflcopter777** [comment]
Worked like a charm, thank you so much ^^

**Comment by Moltdar** [comment]
yup this definetely fixed the reboot issue from https://github.com/ClusterM/hakchi2/issues/495

**Comment by hoderer** [comment]
thanks, but it seems like the portable version isnt updated yet... I downloaded, overwrote my files, kept my preferences and roms settings from last version, flashed custom kernel again, but I still get the C8 error on exit...

EDIT: I got it working. I flashed the kernel via webupdater-version and the games with my old portable. Worked like a charm! Now my SNESC can be plugged back into the usb-port of my TV without turning off! THANKS!

EDIT 2: Forget what I just wrote. Things were getting dirty. Games didn't start. C8 error when trying to start a game and so on. It worked only a second. I re-flashed the kernel with my old (recent) portable and went back. At least games are starting now... Still hoping for a fix.
